dogs.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="mystyle.css">
+</head>
+<body>
+
+  <div class="header">
+    <h1>My Website</h1>
+    <img src="./images/adoptadog-logo.svg" alt="Logo" width="1500" height="800">
+    <a href="./forms.html" class="button">Checkout</a>
+    <p>A website created by me.</p>
+  </div>
+
+<div class="navbar">
+  <a href="index.html">Homepage</a>
+  <a href="./dogs.html">Dogs</a>
+  <a href="./blog.html">Blog</a>
+</div>
+<main>
+  <h2>Here are all our Dogs</h2>
+  <div class="row">
+    <div class="column">
+      <img src="./images/murphy-card.jpg" alt="Murphy">
+      <div class="Murphy">
+        <h2>Murphy</h2>
+        <h3>Cost to Adopt: $123.45</h3>
+        <p>Some text</p>
+      </div>
+    </div>
+   </div>
+
+    <div class="column">
```

Comment:
You're on the right path with grouping these dog tiles, but don't think of them as columns, as they might change  how many tiles are in a row depending on mobile sizes. Think of them as a tile and then worry about laying them out, in some sort of wrapper.